vm/systemSmartContracts/auction.go

```diff
@@ -1034,8 +1076,8 @@ func shuffleList(list []string, random []byte) {
 func isNumArgsCorrectToStake(args [][]byte) bool {
 	maxNodesToRun := big.NewInt(0).SetBytes(args[0]).Uint64()
 	areEnoughArgs := uint64(len(args)) >= 2*maxNodesToRun+1
-
-	return areEnoughArgs
+	areNotTooManyArgs := uint64(len(args)) <= 2*maxNodesToRun+1+2
```

Comment:
use a var with a description for this: 2*maxNodesToRun+1+2

Reply:
added comment.